docs/userguide/dockervolumes.md

```diff
@@ -102,6 +102,10 @@ This will mount the host directory, `/src/webapp`, into the container at
 > If the path `/opt/webapp` already exists inside the container's image, its
 > contents will be replaced by the contents of `/src/webapp` on the host to stay
 > consistent with the expected behavior of `mount`
+>
+> When using Boo2Docker on Windows through git bash, there might be an issue with the 
+> way the source directory name is parsed. You can fix it by using a double slash at
```

Comment:
I fixed the typo.
About the quotes, they didn't work in my case, so I'm not sure they are
relevant. I think the link to the issue should be enough reference for this
alternative approach.

On Fri, Jul 3, 2015 at 1:03 AM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/userguide/dockervolumes.md
> <https://github.com/docker/docker/pull/14235#discussion_r33830773>:
>
> > @@ -102,6 +102,10 @@ This will mount the host directory, `/src/webapp`, into the container at
> >  > If the path `/opt/webapp` already exists inside the container's image, its
> >  > contents will be replaced by the contents of `/src/webapp` on the host to stay
> >  > consistent with the expected behavior of `mount`
> > +>
> > +> When using Boo2Docker on Windows through git bash, there might be an issue with the
> > +> way the source directory name is parsed. You can fix it by using a double slash at
>
> Reading #12751 (comment)
> <https://github.com/docker/docker/issues/12751#issuecomment-109035095> it
> looks like using quotes ' also works in some cases, perhaps we should
> include that as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14235/files#r33830773>.
>